### PR TITLE
[DOCFIX]Update "Browse File System" to "Browse"

### PR DESCRIPTION
**Pull request by ganganFighting**
https://alluxio.atlassian.net/browse/ALLUXIO-2006
ALLUXIO-2006
Update "Browse File System" to "Browse" in docs/en/Web-Interface.md to be consistent with the Web UI navigation bar

### CODE REVIEW

**Comment by ifcharming** [comment]
ok to test

File: docs/en/Web-Interface.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The configuration page has two sections:
     made to a file not prefixed by a path in the white list. Only whitelisted files will be stored
     in Alluxio.
 
-## Browse File System Page
+## Browse
```
Comment (ifcharming) [inline docs/en/Web-Interface.md]: you might want to update `Browse File System` to `Browse` on line 67 instead of the header here?

**Comment by ifcharming** [comment]
thanks @ganganFighting for the contribution, I left a comment.

File: docs/en/Web-Interface.md (hunk 1 above)
Comment (ganganFighting) [inline docs/en/Web-Interface.md]: Sorry, stupid of me. I have fix this and re-committed. :-)

Comment (ifcharming) [inline docs/en/Web-Interface.md]: I meant you should keep "Browse File System Page" here.
And update line 67:
When selecting the "Browse File System" tab -> When selecting the "Browse" tab

Comment (ganganFighting) [inline docs/en/Web-Interface.md]: Ok, I fix both and re-committed. Thank you~

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**